### PR TITLE
Post like comments

### PR DESCRIPTION
**Pull request by JatuFaique**
feat:
added like and dislike api calls on posts
add and deletes comments by user

### CODE REVIEW

File: src/Components/PostCard.js [hunk 1]
```diff
@@ -1,67 +1,132 @@
...
+
+  const handleCommentSubmit = () => {
+    dispatch(
+      userComments({ token: token, commentContent: comment, postId: post._id })
+    );
+    setComment({ text: "" });
+  };
+
+  const handleCommentDelete = (commentId) => {
+    console.log("ybddbb", commentId);
```
Comment (mansipadgelwar) [inline src/Components/PostCard.js:30]: Remove console log

File: src/Components/PostCard.js [hunk 2]
```diff
@@ -1,67 +1,132 @@
...
+        <div className="post__text py-1">
           <p>{post.content}</p>
         </div>
-        <div class="post__buttons flex py-1">
-          <span class="txt-scn">
-            {/* <!-- <i class="fa-solid fa-heart"></i> --> */}
-            <i class="fa-regular fa-heart"></i>
+        <div className="post__buttons flex py-1">
+          <span onClick={likeHandler} className="btn txt-scn">
+            {/* <!-- <i className="fa-solid fa-heart"></i> --> */}
```
Comment (mansipadgelwar) [inline src/Components/PostCard.js:66]: remove

File: src/Components/PostCard.js [hunk 3]
```diff
@@ -1,67 +1,132 @@
...
+                    <i class=" btn trash fa-solid fa-trash"></i>
+                  </span>
+                ) : (
+                  <></>
+                )}
               </div>
             );
           })}
 
-          {/* <div class="comment__card border-radius p-0-5">
```
Comment (mansipadgelwar) [inline src/Components/PostCard.js:56]: remove

File: src/features/auth/authSlice.js [hunk 4]
```diff
@@ -58,7 +58,7 @@ export const authSlice = createSlice({
     },
     [authHandler.fulfilled]: (state, action) => {
       console.log("ho gayi", action.payload);
```
Comment (mansipadgelwar) [inline src/features/auth/authSlice.js:60]: remove

**Review by mansipadgelwar — APPROVED** [review]
You can merge.

File: src/features/timeline/postSlice.js [hunk 5]
```diff
@@ -39,6 +39,104 @@ export const userPosts = createAsyncThunk(
...
+        {
+          headers: {
+            authorization: payload.token,
+          },
+        }
+      );
+      console.log(res.data);
+      return res.data;
+    } catch (error) {
+      console.log(error);
```
Comment (kgajriaAgarwal) [inline src/features/timeline/postSlice.js:60]: use Toast for showing the errors

File: src/features/timeline/postSlice.js [hunk 6]
```diff
@@ -39,6 +39,104 @@ export const userPosts = createAsyncThunk(
...
+      const res = await axios.post(
+        `/api/posts/dislike/${payload.postId}`,
+        {},
+        {
+          headers: {
+            authorization: payload.token,
+          },
+        }
+      );
+      console.log(res.data);
```
Comment (kgajriaAgarwal) [inline src/features/timeline/postSlice.js:79]: remove console.log

File: src/features/timeline/postSlice.js [hunk 7]
```diff
@@ -39,6 +39,104 @@ export const userPosts = createAsyncThunk(
...
+      return res.data;
+    } catch (error) {
+      console.log(error);
```
Comment (kgajriaAgarwal) [inline src/features/timeline/postSlice.js:82]: remove console.log

File: src/features/timeline/postSlice.js [hunk 8]
```diff
@@ -39,6 +39,104 @@ export const userPosts = createAsyncThunk(
...
+      return rejectWithValue(error);
+    }
+  }
+);
+
+export const userDeleteComments = createAsyncThunk(
+  "post/userDeleteComments",
+  async (payload, { rejectWithValue }) => {
+    console.log("idhr", payload.commentId);
```
Comment (kgajriaAgarwal) [inline src/features/timeline/postSlice.js:117]: remove console.log

**Review by kgajriaAgarwal — CHANGES_REQUESTED** [review]
Remove Console.log, rest everything is fine and can be merged

**Pull request merged**